src/components/img/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import Skeleton from 'react-loading-skeleton';
+
+export default class Img extends Component {
+  refImg = React.createRef();
+
+  state = {
+    isLoading: true,
+  };
+
+  componentDidMount() {
+    const image = this.refImg.current;
+    image.addEventListener('load', this.onLoad);
+    image.addEventListener('error', this.onError);
+  }
+
+  componentWillUnmount() {
+    const image = this.refImg.current;
+    image.removeEventListener('load', this.onLoad);
+    image.removeEventListener('error', this.onError);
+  }
+
+  onLoad = () => {
+    this.setState({
+      isLoading: false,
+    });
+  };
+
+  onError = () => {
+    const image = this.refImg.current;
+    this.setState({
+      isLoading: false,
+    });
+    image.src = 'http://placeimg.com/640/480/tech';
+  };
```

Comment:
Looks like hardcode. Move it to constants or download this picture to assets

Reply:
done